includes/supervisor/supervisord.conf

```diff
@@ -9,94 +9,94 @@ autorestart = true
 environment=HOME="/home/healthchecks",USER="healthchecks"
 stdout_events_enabled = true
 stderr_events_enabled = true
-stdout_logfile = /var/log/gunicorn_stdout.log
-stdout_logfile_maxbytes = 2MB
-stderr_logfile = /var/log/gunicorn_stderr.log
-stderr_logfile_maxbytes = 2MB
+stdout_logfile = /dev/fd/1
```

Comment:
Did that work for you for all the processes?
I think I had experimented with it some time ago but I wasn't able to get it working.

Reply:
You are right for smtpd, sendalerts and sendreports it doesn't work like this, because python buffers stdout.
However, logs are not written to /var/log for these processes either (probably it is delayed due to buffering and I was too impatient while testing).

Reply:
I pushed a change to run Python in unbuffered mode, then it works for all processes.
I also changed it to /dev/std[out,err], same function but it reads better.